src/map/skill.cpp

```diff
@@ -7647,9 +7720,16 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 	case TR_MYSTIC_SYMPHONY:
 	case TR_KVASIR_SONATA:
 	case EM_SPELL_ENCHANTING:
+	case NPC_GRADUAL_GRAVITY:
+	case NPC_DAMAGE_HEAL:
```

Comment:
From my test using your PR, the monster using NPC_IMMUNE PROPERTY + NPC_DAMAGE_HEAL is still healed by a positive number. No idea how it works on KRO, so for me the heal should be 0

Reply:
My bad. Fixed.